packages/modelviewer.dev/data/examples.json

```diff
@@ -13,15 +13,15 @@
       },
       {
         "htmlId": "demo-container-3",
-        "name": "Customize Loading"
+        "name": "Customize Load"
       },
       {
         "htmlId": "demo-container-4",
-        "name": "Poster with invalid model"
+        "name": "Invalid Model"
       },
       {
         "htmlId": "demo-container-5",
-        "name": "Cycling between posters"
+        "name": "Cycling Posters"
```

Comment:
Let's make separate sections for static methods and static properties. This will let you remove `ModelViewer.` from those, shortening them considerably. Let's see if that allows us to narrow the sidebar.

Reply:
done